data_model/src/isi.rs

```diff
@@ -903,14 +742,178 @@ impl NewParameterBox {
         self.parameter.len() + 1
     }
 
-    /// Construct [`SetParameterBox`].
+    /// Construct [`NewParameterBox`].
     pub fn new<P: Into<EvaluatesTo<Parameter>>>(parameter: P) -> Self {
         Self {
             parameter: parameter.into(),
         }
     }
 }
 
+pub mod error {
+    //! Module containing errors that can occur during instruction evaluation
+
+    use iroha_primitives::fixed::FixedPointOperationError;
+
+    use super::*;
+    use crate::{
+        metadata,
+        query::error::{FindError, QueryExecutionFailure},
+    };
+
+    model! {
+        /// Instruction execution error type
+        #[derive(Debug, Display, FromVariant)]
+        #[cfg_attr(feature = "std", derive(thiserror::Error))]
+        // TODO: Only temporarily opaque because of InstructionExecutionFailure::Repetition
+        #[ffi_type(opaque)]
+        pub enum InstructionExecutionFailure {
+            /// Failed to find some entity
+            #[display(fmt = "Failed to find. {_0}")]
+
```

Comment:
I wonder why `rustfmt` won't remove this empty new line.

Reply:
`rustfmt` is fairly limited in what it changes. For example it doesn't convert tabs into spaces unless you tell it to do so (and for good reason). 

In most cases we assume that blank lines have a meaning, and help with readability. In this case it seems like a mistake.